src/clarity-angular/wizard/providers/button-hub.ts

```diff
@@ -10,6 +10,14 @@ import {Subject} from "rxjs/Subject";
 
 @Injectable()
 export class ButtonHubService {
+    private _buttonsReady: boolean = false;
```

Comment:
I am curious why this property couldn't be just a public property?

Reply:
It was originally an observable, but then I solved it without and left the getter setting. Will just make it public.